internal/clients/recommendations/recommendations.v3.json

```diff
@@ -120,6 +126,26 @@
           "recommendedPackages": {
             "type": "integer",
             "format": "int32"
+          },
+          "distribution": {
+            "$ref": "#/components/schemas/DistributionItem"
+          }
+        }
+      },
+      "DistributionItem": {
+        "type": "object",
+        "required": [
+          "name",
+          "description"
+        ],
+        "properties": {
+          "description": {
+            "type": "string",
+            "example": "Red Hat Enterprise Linux (RHEL) 8.4"
+          },
+          "name": {
+            "type": "string",
+            "example": "rhel-84"
```

Comment:
Do I understand this correctly that description a required item for a recommendation request? This does not make any sense, the name is unique enough.